docs/js/files/modules_FormModel.js.html

```diff
@@ -380,12 +380,24 @@ <h1 class="file-heading">File: modules/FormModel.js</h1>
 
     /**
      * Update or create a binding between an object model and an alias.
-     * @method setTrackedModel
+     * @method trackModel
      * @param alias {String} the alias/name to bind to.
      * @param model {Backbone Model instance} the model to be bound. Mappings referencing this alias will start applying to this model.
      * @param [copy=false] {Boolean} if true, the form model will perform a pull on any mappings using this alias.
+     * @deprecated use trackModel instead.
      */
-    setTrackedModel: function(alias, model, copy) {
+    setTrackedModel: function() {
```

Comment:
oh man, i should have said the first time to reference the newer method's javadoc instead of copying pasting it. up to you if you want to update it or merge it.